_config.yml

```diff
@@ -78,11 +78,11 @@ optional_components:
 
 source_releases:
   -
-      name: "Apache Flink 1.7.1"
-      id: "171-download-source"
-      url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.7.1/flink-1.7.1-src.tgz"
-      asc_url: "https://www.apache.org/dist/flink/flink-1.7.1/flink-1.7.1-src.tgz.asc"
-      sha512_url: "https://www.apache.org/dist/flink/flink-1.7.1/flink-1.7.1-src.tgz.sha512"
+      name: "Apache Flink 1.7.2"
+      id: "172-download-source"
+      url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.7.2/flink-1.7.2-src.tgz"
+      asc_url: "https://www.apache.org/dist/flink/flink-1.7.2/flink-1.7.2-src.tgz.asc"
+      sha512_url: "https://www.apache.org/dist/flink/flink-1.7.2/flink-1.7.2-src.tgz.sha512"
   -
       name: "Apache Flink 1.6.3"
```

Comment:
1.7.2？

Reply:
That is supposed to be an entry for 1.6.3.